templates/server/puppetserver/bootstrap.cfg.erb

```diff
@@ -0,0 +1,15 @@
+puppetlabs.services.request-handler.request-handler-service/request-handler-service
+puppetlabs.services.jruby.jruby-puppet-service/jruby-puppet-pooled-service
+puppetlabs.services.puppet-profiler.puppet-profiler-service/puppet-profiler-service
+puppetlabs.trapperkeeper.services.webserver.jetty9-service/jetty9-service
+puppetlabs.trapperkeeper.services.webrouting.webrouting-service/webrouting-service
+puppetlabs.services.config.puppet-server-config-service/puppet-server-config-service
+puppetlabs.services.master.master-service/master-service
+puppetlabs.services.puppet-admin.puppet-admin-service/puppet-admin-service
+puppetlabs.services.version.version-check-service/version-check-service
+
```

Comment:
I think this OK as-is if it works with 1.x and 2.x, Otherwise I'd recommend rather augeas, as it's a tad more predictable/idempotent than `file_line` in such cases, IMHO.